helm/charts/hydra/values.yaml

```diff
@@ -376,3 +376,11 @@ pdb:
   enabled: false
   spec:
     minAvailable: 1
+
+serviceMonitor:
```

Comment:
Since this is a shared setting structure, which will be shared globally, across all charts, could you also update the rest of them (keto, oathkeeper) to also have their monitors configurable?

Reply:
and done for [#428](https://github.com/ory/k8s/issues/428) ;)